roles/matrix-postgres-backup/tasks/util/detect_existing_postgres_version.yml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
This file should probably be completely deleted.

Reply:
This file is used in setup_postgres_backup.yml to detect the currently used postgres version. The detected version is used to download/use the corresponding docker image for the prodrigestivill/docker-postgres-backup-local container.